pyccl/tests/test_tkkssc.py

```diff
@@ -23,6 +27,34 @@
 AA = 1.0
 
 
+def get_ssc_counterterm_gc(k, a, hmc, prof1, prof2, prof12_2pt,
+                           normalize=False):
+
+    P_12 = b1 = b2 = np.zeros_like(k)
+    if prof1.is_number_counts or prof2.is_number_counts:
+        norm1 = hmc.profile_norm(COSMO, a, prof1)
+        norm2 = hmc.profile_norm(COSMO, a, prof2)
+        norm12 = 1
+        if prof1.is_number_counts:
```

Comment:
I don't understand this nested if clause. If `hmc.profile_norm()` works well regardless of `prof1.is_number_counts` or `prof2.is_number_counts` you can just do `norm12 = norm1 * norm2`, do I understand this right?